src/core/modules/Nebula.cpp

```diff
@@ -967,11 +980,13 @@ void Nebula::drawHints(StelPainter& sPainter, float maxMagHints, StelCore *core)
 			renderDarkNebulaMarker(sPainter, XY[0], XY[1], finalSize, col);
 			return;
 		case NebCn:
-			if (finalSize > 35.f)
-				Nebula::texOpenClusterWithNebulosityLarge->bind();
-			else
-				Nebula::texOpenClusterWithNebulosity->bind();
-			break;
+		{
+			renderMarkerRoundedRect(sPainter, XY[0], XY[1], finalSize, col);
+			if(objectInDisplayedType())
```

Comment:
I think this test is not required. The symbol should be drawn completely or not at all. 

With that bicolor icon, we can and should remove `NebulaMgr::clusterWithNebulosityColor` etc., as it should be combined from openClustersColor and NebulaColor.

Reply:
This test is taken from line 913. If it is to be removed here, the same should be done there.

Reply:
Uh, line 913 is funny. It should not set black, but I think it could simply return at this point. @alex-w ?

Reply:
@gzotti within `drawHints()` ?

Reply:
@alex-w [here](https://github.com/Stellarium/stellarium/pull/3172/files/0e9d3c620bb3542d76763e88696df64f609c836e#diff-e081b39a4a7e50860a2c73cfa76ccb2d5ba2df7fb09c7b716a96ab096f11b33fL913)

Reply:
Yes, it should be `return;` here

Reply:
> I think this test is not required. The symbol should be drawn completely or not at all.
> 
> With that bicolor icon, we can and should remove `NebulaMgr::clusterWithNebulosityColor` etc., as it should be combined from openClustersColor and NebulaColor.

Hmm... in this case the UI for DSO color and config data should be updated too.

Reply:
Yes, of course. Removing one color will even look better in the current UI.